packages/expo-updates/android/src/main/java/expo/modules/updates/db/UpdatesDatabase.kt

```diff
@@ -161,5 +162,23 @@ abstract class UpdatesDatabase : RoomDatabase() {
         }
       }
     }
+
+    val MIGRATION_8_9: Migration = object : Migration(8, 9) {
+      override fun migrate(database: SupportSQLiteDatabase) {
+        // https://www.sqlite.org/lang_altertable.html#otheralter
+        try {
+          database.execSQL("PRAGMA foreign_keys=OFF")
+          try {
+            database.beginTransaction()
```

Comment:
Not super important, but @jkhales 's suggestion here is confusing to me 🤔 The outer finally block would turn foreign keys on even if `beginTransaction` threw an exception in the outer try block. The way this is written now, if there is an error executing `beginTransaction` the inner finally block will still run and execute `endTransaction` -- which seems more wrong? Or am I missing/misreading something here??

Reply:
Reverting the change. Yep, this makes sense to me.